src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -1,8 +1,7 @@
 package core.mate.academy.model;
 
 /**
- * Add some custom fields that could be only in Truck
- * Do not remove no-field constructor
+ * Add some custom fields that could be only in Truck Do not remove no-field constructor
```

Comment:
The comment suggests adding custom fields specific to a Truck. Consider adding fields such as `cargoCapacity`, `numberOfAxles`, or other Truck-specific attributes.